### PR TITLE
Feature/in numeric cols

### PR DESCRIPTION
**Pull request by lwolters**
Fixed issue with tokenizing NONE clauses

### CODE REVIEW

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala [hunk 1]
```diff
@@ -7,39 +7,37 @@ trait LogicalFunctionTokenizer {
 
   def tokenizeLogicalFunction(col: LogicalFunction)(implicit ctx: TokenizeContext): String =
     (col.left.asOption, col.right.asOption) match {
-      case (None, None)        => "1"
-      case (Some(left), None)  => tokenizeColumn(left)
-      case (None, Some(right)) => tokenizeColumn(right)
+      case (None, None)       => "1"
+      case (Some(left), None) => tokenizeColumn(left)
```
Comment (sjoerdmulder) [inline dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:11]: Should be the same as `(None, Some(right))` but then with left?

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala [hunk 2]
```diff
@@ -7,39 +7,37 @@ trait LogicalFunctionTokenizer {
...
+        if (right.isConstTrue) tokenizeColumn(right)
+        else tokenize(right, col.operator)
       case (Some(left), Some(right)) =>
         col.operator match {
           case And =>
-            if (left.isConstTrue)
-              tokenizeColumn(right)
-            else if (right.isConstTrue)
-              tokenizeColumn(left)
+            if (left.isConstTrue) tokenizeColumn(right)
```
Comment (sjoerdmulder) [inline dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:18]: Use `tokenize`?

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala [hunk 3]
```diff
@@ -7,39 +7,37 @@ trait LogicalFunctionTokenizer {
...
             else {
               // Depending on the number of clauses (to the right or left) we should add parentheses/brackets or not
               s"${tokenize(left, And)} AND ${tokenize(right, And)}"
             }
           case Or =>
-            if (left.isConstFalse)
-              tokenizeColumn(right)
-            else if (right.isConstFalse)
-              tokenizeColumn(left)
+            if (left.isConstFalse) tokenizeColumn(right)
```
Comment (sjoerdmulder) [inline dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:25]: use `tokenize`?

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala [hunk 4]
```diff
@@ -7,39 +7,37 @@ trait LogicalFunctionTokenizer {
...
               // Depending on the number of clauses (to the right or left) we should add parentheses/brackets or not
               s"${tokenize(left, Or)} OR ${tokenize(right, Or)}"
             }
-          case Xor =>
-            s"xor(${tokenizeColumn(left)}, ${tokenizeColumn(right)})"
-          case Not =>
-            s"not(${tokenizeColumn(left)})"
+          case Xor => s"xor(${tokenizeColumn(left)}, ${tokenizeColumn(right)})"
+          case Not => s"not(${tokenizeColumn(left)})"
+//          case Xor => s"xor(${tokenize(left, Xor)}, ${tokenize(right, Xor)})"
```
Comment (sjoerdmulder) [inline dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:33]: Remove old comments

File: dsl/src/test/scala/com/crobox/clickhouse/dsl/language/LogicalFunctionTokenizerTest.scala [hunk 5]
```diff
@@ -125,9 +125,17 @@ class LogicalFunctionTokenizerTest extends ClickhouseClientSpec with TestSchema
     )
   }
 
+  it should "tokenize numbers OR with NONE" in {
```
Comment (sjoerdmulder) [inline dsl/src/test/scala/com/crobox/clickhouse/dsl/language/LogicalFunctionTokenizerTest.scala:128]: Add queries that hit the `isConstFalse` and `isConstTrue` codepaths?

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala (hunk 1 above)
Comment (lwolters) [reply dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:11]: Enabling it for the left side is not required. I did enabled it, but some other tests failed

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala (hunk 2 above)
Comment (lwolters) [reply dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:18]: Well, the clause is a constant, so why putting (possible) brackets around it?

File: dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala (hunk 4 above)
Comment (lwolters) [reply dsl/src/main/scala/com.crobox.clickhouse/dsl/language/LogicalFunctionTokenizer.scala:33]: will do

File: dsl/src/test/scala/com/crobox/clickhouse/dsl/language/LogicalFunctionTokenizerTest.scala (hunk 5 above)
Comment (lwolters) [reply dsl/src/test/scala/com/crobox/clickhouse/dsl/language/LogicalFunctionTokenizerTest.scala:128]: Done

**Pull request merged**